### ISSUE TITLE
Accessing solver parameters

### ISSUE THREAD

**Issue by engharat** [NVIDIA/DIGITS#730]
Hi, 
I have seen some proposals on fully accessing the solver.prototxt before starting the training (and that would be the best solution I think). In the meanwhile, I don't see anywhere how could we access some hyperparameters like ADAM eps parameter or RMSPROP rms_decay .i see on the caffe_train.py the weight decay and the moment parameters, but I cannot find and hints to where to find and where to edit the default ADAM, ADADELTA  and RMSPROP parameters.

**Comment by lukeyeager** [comment]
To edit the defaults manually, you'll want to make changes to this section:
https://github.com/NVIDIA/DIGITS/blob/v3.3.0/digits/model/tasks/caffe_train.py#L491-L501

You could add something like:
```py
if solver.solver_type == solver.ADAM:
  solver.eps = MY_DEFAULT
```
For more control, see #565.

**Comment by lucbettaieb** [comment]
It would be cool to see this as a front-end feature.
